### DISCUSSION TITLE
fancybox 5: how to do caption below the image, instead of overlapping

### DISCUSSION THREAD

**Discussion by mDesignHub** [fancyapps/ui#640] [General]
fancybox 5: is there a way to do caption below the image on small screens (i.e mobile), instead of overlapping. at the moment my image is completely overlapped by a longer caption.

**Comment by fancyapps** [comment]
Hi,

v5 is a "compact mode" that tries to mimic iOS Photos functionality for small screens. You can disable it using the `compact` option:
https://jsfiddle.net/a8hu4mor/

**Reply by mDesignHub** [reply]
Ah :) this is great. you have thought of everything ;) thank you!
